.github/CONTRIBUTING.md

```diff
@@ -21,3 +23,20 @@ If you don't have any particular package in mind to start with:
 - [Check roadmap](https://github.com/yiisoft/docs/blob/master/003-roadmap.md).
 - Check package issues at github. Usually there are some.
 - Ask @samdark.
+
+## Reporting a Security Issue
+
+If you have encountered a security-related issue with Yii, please use the [security issue form](https://www.yiiframework.com/security) to report any security issues you have encountered to us.
+
+***WARNING!*** DO NOT use the issue tracker or discuss it in the public forum as it will cause more damage than help.
+
+# Additional Resources
+
+- [Yii coding standards](https://github.com/yiisoft/docs/blob/master/010-code-style.md)
+- [Forum](https://www.yiiframework.com/forum/)
+- [Yii3 Existing issues](https://forum.yiiframework.com/c/yii-3-0/63)
```

Comment:
This is forum, not issues.

Reply:
remove "[Forum](https://www.yiiframework.com/forum/)" ?
add something else?

Reply:
Seems, "Yii3 Existing issues" is a invalid label.